src/silvimetric/cli/cli.py

```diff
@@ -33,10 +34,22 @@ def cli(ctx, database, debug, log_level, log_dir, progress):
 
 
 @cli.command("info")
-@click.option("--history", is_flag=True, default=False, type=bool)
+@click.option("--bounds", type=BoundsParamType(), default=None)
+@click.option("--date", type=click.DateTime(['%Y-%m-%d','%Y-%m-%dT%H:%M:%SZ']), multiple=True, default=None)
+@click.option("--name", type=str, default=None)
 @click.pass_obj
-def info_cmd(app, history):
-    info.info(app.tdb_dir, history)
+def info_cmd(app, bounds, date, name):
+    import numpy as np
+    if len(date) == 1:
```

Comment:
Is there a date parsing function in `datetime` we can use here? It would be better for users to provide dates in standard formats

Reply:
There is, but I was finding numpy's datetime usage to be easier to use. Can go back to normal datetime library though if necessary.